### ISSUE TITLE
'smooth: true' causes scrollTo to not work in Chrome, Edge, but works on Firefox

### ISSUE THREAD

**Issue by cupof2** [fisshy/react-scroll#490]
```
scroller.scrollTo(scrollRef.current[nextPage].props.name, {
                    duration: 800,
                    delay: 0,
                    smooth: true
                });
```
doesn't work in chrome and edge. works on firefox though.

When I remove smooth: true, it works on every browser. but it's ugly

**Comment by Deadpool957** [comment]
I have the same problem，when I use smooth，Cannot roll to specified position

**Comment by fisshy** [comment]
Could you provide a plunkr/codepen to demonstrate the issue?

**Comment by Deadpool957** [comment]
https://codesandbox.io/s/container-forked-c3kse?file=/src/index.js:0-2586

**Comment by Deadpool957** [comment]
it can not work

**Comment by Deadpool957** [comment]
Roll inside the container，The height of the container is fixed, and the page will scroll automatically after loading

**Comment by Deadpool957** [comment]
you can see this,https://codesandbox.io/s/container-forked-c3kse?file=/src/index.js

**Comment by fisshy** [comment]
@rjworks Would it work if you wrote it like this instead.

```
scroller.scrollTo("scroll-to-element", {
      duration: 800,
      delay: 0,
      smooth: "easeInOutQuart",
      offset: offset
    });
```

**Comment by Calisto184AC0** [comment]
I had the same problem. But in my case I had a CSS property like this:
`html { scroll-behavior: smooth }`
When I removed this line it worked correctly.

**Comment by saschadiercks** [comment]
Yes, it seems react-scroll doesn't work as expected when the `scroll-behavior` is set.
Since it is not always an option, to remove the scroll-behavior from the CSS (because it is quite handy) - is there a way to work around this using react-scroll?

**Comment by danny-dang** [comment]
For me, `scroll-behavior: smooth` also caused the issue.

Maybe it's better to have this documented.

**Comment by sbuys** [comment]
> For me, `scroll-behavior: smooth` also caused the issue.
> 
> Maybe it's better to have this documented.

Yes please update the documentation with this. Would have saved team many hours.
